app/lib/submission_builder/authentication_header.rb

```diff
@@ -7,6 +7,13 @@ def document
       build_xml_doc("AuthenticationHeader") do |xml|
         xml.FilingLicenseTypeCd "O"
         xml.FinancialResolution do
+          if @intake.has_banking_information_in_financial_resolution? && @intake.payment_or_deposit_type == "direct_deposit"
+            xml.FirstInput do
+              xml.RoutingTransitNum sanitize_for_xml(@intake.routing_number) if @intake.routing_number.present?
+              xml.DepositorAccountNum sanitize_for_xml(@intake.account_number) if @intake.account_number.present?
+              xml.InputTimestamp @intake.primary_esigned_at.in_time_zone(StateFile::StateInformationService.timezone("md")).strftime("%FT%T%:z") if @intake.primary_esigned_at.present?
```

Comment:
found the `strftime` format needed to satisfy xsd schema https://apidock.com/ruby/DateTime/strftime